### PR TITLE
[Challenge Platform] Clarify browser extension compatibility with Turnstile

### PR DESCRIPTION
**Pull request by patriciasantaana**
PCX-10998

### CODE REVIEW

File: content/waf/reference/cloudflare-challenges.md [hunk 1]
```diff
@@ -193,5 +189,7 @@ Cross-origin resource sharing (CORS) preflight requests, or `OPTIONS`, exclude u
 
 Cloudflare challenges cannot support the following:
 
+* Random canvas fingerprinting.
+* [Browser extensions](#browser-extensions) that modify your browser’s default `User-Agent` value.
```
Comment (patriciasantaana) [inline content/waf/reference/cloudflare-challenges.md]: @migueldemoura duplicated the extensions UA info into the limitations section. wdyt?

**Comment by patriciasantaana** [comment]
#14732

File: content/waf/reference/cloudflare-challenges.md (hunk 1 above)
Comment (migueldemoura) [reply content/waf/reference/cloudflare-challenges.md]: Sure, challenge pages are mostly Turnstile, so this applies to both sections.

Comment (migueldemoura) [inline content/waf/reference/cloudflare-challenges.md]: I'd remove the above note and replace this one with something like:

```
* [Browser extensions](#browser-extensions) that modify the browser's `User-Agent` value or Web APIs like `Canvas` and `WebGL`.
```

**Review by migueldemoura — CHANGES_REQUESTED** [review]

File: content/waf/reference/cloudflare-challenges.md [hunk 2]
```diff
@@ -193,5 +187,7 @@ Cross-origin resource sharing (CORS) preflight requests, or `OPTIONS`, exclude u
 
 Cloudflare challenges cannot support the following:
 
+* Random canvas fingerprinting.
+* [Browser extensions](#browser-extensions) that modify the browser's `User-Agent` value or Web APIs such as `Canvas` and `WebGL`.
```
Comment (patriciasantaana) [inline content/waf/reference/cloudflare-challenges.md:190]: @migueldemoura removed the note above and modified this bullet as requested

**Review by migueldemoura — APPROVED** [review]

**Pull request merged**